pbs_light.go

```diff
@@ -885,6 +885,7 @@ func serve(cfg *config.Configuration) error {
 		OptOutURL:    cfg.HostCookie.OptOutURL,
 		OptInURL:     cfg.HostCookie.OptInURL,
 		OptOutCookie: cfg.HostCookie.OptOutCookie,
+		TTL:          time.Duration(cfg.HostCookie.TTL) * 24 * time.Hour,
```

Comment:
Are you sure this is right?

It looks like you're multiplying by `* 24 * time.Hour` both here and in `ToHTTPCookie` in `usersync.go`. I'm not certain, but... seems like a bug?

Reply:
Good catch. go is fine implicitly converting a static `int` to `time.Duration`, but not converting a variable of type `int64`. So I found I had to cascade back a `time.Duration`declaration, and at the end (beginning) I again did a day-time conversion. I think its cleaner to do the conversion at the front, and let `ttl` be a `time.duration` throughout the code than do it as the last step.

Reply:
I had almost exactly the same bug recently. I don't like how Go implements these type aliases at all.

I think your strategy is the right one, though. Convert once ASAP, and pass the value through from there.